code/workspaces/web-app/src/selectors/dataStorageSelectors.js

```diff
@@ -0,0 +1,5 @@
+const selectDataStoraageArray = ({ dataStorage }) => dataStorage;
```

Comment:
Type in Storage in `selectDataStoraageArray`

Reply:
Type faxed :-)